package.json

```diff
@@ -3,20 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "lint": "next lint",
-    "tina-dev": "yarn tinacms server:start -c \"next dev\"",
-    "tina-build": "yarn tinacms server:start -c \"next build\"",
-    "tina-start": "yarn tinacms server:start -c \"next start\""
+    "dev": "yarn tinacms server:start -c \"next dev\"",
```

Comment:
Before when running `yarn dev` and navigating to the blog page an error would come up. If we run the graphQL server no error will come up.